config.d

```diff
@@ -48,11 +48,13 @@ Returns: the generated version, or the content of `versionFile`
 */
 string generateVersion(const string versionFile)
 {
-    import std.process : execute;
+    import std.process : Config, environment, execute;
     import std.file : readText;
     import std.string : strip;
 
-    const result = execute(["git", "describe", "--dirty"]);
+    const workingDirectory = environment.get("DUB_PACKAGE_DIR", ".");
```

Comment:
`__FILE_FULL_PATH__` would probably be even safer.

Reply:
Fixed.